_content/av/buddhist-spirituality-and-contemplation-through-poetry_hallisey-charles.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Buddhist Spirituality and the Contemplation of Nature Through Poetry"
+authors:
+  - hallisey-charles
+subcat: poetry
```

Comment:
Oh yeah, thanks for pointing that out. Missed this the first time. `subcat: poetry` is for things that **are** poetry. The poetry tag is for things **about** poetry. Same with, e.g. `film`. `subcat: film` is for movies, `tags: [film]` is for e.g. essays about the silver screen.